apps/test/unit/gamelab/GameLabGroupTest.js

```diff
@@ -33,4 +34,237 @@ describe('GameLabGroup', function () {
       expect(sprite2.setSpeedAndDirection.calledWith(speed, direction)).to.be.true;
     });
   });
+
+  describe('collision methods', function () {
+    var sprite1, sprite2, targetSprite, group;
+
+    beforeEach(function () {
+      sprite1 = createSprite(0, 0, 100, 100);
+      sprite2 = createSprite(400, 400, 100, 100);
+      targetSprite = createSprite(200, 200, 100, 100);
+      group = createGroup();
+      group.add(sprite1);
+      group.add(sprite2);
+    });
+
+    describe('no sprite in group overlaps target sprite', function () {
+      it('returns false for isTouching', function () {
+        let result;
+        for (let i = 0; i < group.length + 1; i++) {
+          result = group.isTouching(targetSprite);
+        }
```

Comment:
I see. This test seems to know a lot about the implementation of the things it is testing. I see that the only level test for gamelab is commented out: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/test/integration/levelSolutions/gamelab/custom.js 

OK to not fix level tests right now, but once you have them working it might allow you to test things like this is a more end-to-end fashion.